src/gui.js

```diff
@@ -5412,20 +5412,20 @@ IDE_Morph.prototype.initializeCloud = function () {
                     myself.source = 'cloud';
                     if (!isNil(response.days_left)) {
                         new DialogBoxMorph().inform(
-                            'Unverified account: ' +
+                            localize('Unverified account: ') +
                             response.days_left +
-                            ' days left',
-                            'You are now logged in, and your account\n' +
-                            'is enabled for three days.\n' +
-                            'Please use the verification link that\n' +
-                            'was sent to your email address when you\n' +
-                            'signed up.\n\n' +
-                            'If you cannot find that email, please\n' +
-                            'check your spam folder. If you still\n' +
-                            'cannot find it, please use the "Resend\n' +
-                            'Verification Email..." option in the cloud\n' +
-                            'menu.\n\n' +
-                            'You have ' + response.days_left + ' days left.',
+                            localize(' days left'),
```

Comment:
hmm... for long strings such as the ones in this PR it might make sense to leave it as a single large one, instead of breaking it up into several substrings. I can imagine some languages have a different sentence structure...

Reply:
@jmoenig
I think long strings can be broken into smaller ones. It is good to avoid long strings, both in code and in translations (translations are also code).

But yes, for me the mistake here is to break the message in lines. We can break long messages in sentences. That will be good to have shorter lines and shorter strings... and also good to take the advantage of the work done: some sentences will be repeated more than once, and they will need only one translation (sentences like "Are you sure?").

I added the "localizes" and I didn't want to change the origin code. But if you want, I will update the PR cutting the messages by sentences.